template/xwiki/docker-entrypoint.sh

```diff
@@ -127,6 +127,9 @@ function configure() {
   safesed "replacepassword" \$DB_PASSWORD /usr/local/tomcat/webapps/\$CONTEXT_PATH/WEB-INF/hibernate.cfg.xml
   safesed "replacecontainer" \$DB_HOST /usr/local/tomcat/webapps/\$CONTEXT_PATH/WEB-INF/hibernate.cfg.xml
   safesed "replacedatabase" \$DB_DATABASE /usr/local/tomcat/webapps/\$CONTEXT_PATH/WEB-INF/hibernate.cfg.xml
+  
+  safesed "xwiki.db=xwiki" "xwiki.db=$DB_DATABASE" /usr/local/tomcat/webapps/\$CONTEXT_PATH/WEB-INF/xwiki.cfg
```

Comment:
Thanks for the PR! :)

In the xwiki.cfg file, I see:

```
#-# [Since 1.6M1]
#-# Force the database name for the main wiki.
# xwiki.db=xwiki
```

Are you sure that using `safesed` will remove the leading comment sign?

Did you try this? Also, when the db is `xwiki` we shouldn't uncomment this property so that the default setup is used.